config/app.php

```diff
@@ -210,6 +210,7 @@
 
     'aliases' => Facade::defaultAliases()->merge([
         // 'ExampleClass' => App\Example\ExampleClass::class,
```

Comment:
Yuck

Reply:
Is the yuck meant for the usage of the alias?

Reply:
@FrciSmrci just for leaving the commented line of example class in :P